### PR TITLE
Add Max Fee Per Gas to transaction breakdown

### LINKED ISSUE #11879: Show "max fee per gas" in transaction history
Especially for submitted transactions. Could be useful. Should be easy to add just that one field, it's already on every 1559 transaction.

Will be even more useful when we display the current/recent base fee somewhere.

### PR DESCRIPTION
**Pull request by mcmire**
This comes through in EIP-1559 transactions and so it is helpful to show
this to users.

---

Fixes #11879.

Screen Shot 2021-09-08 at 10 50 43 AM

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -2564,10 +2564,13 @@
     "message": "Transaction Fee"
   },
   "transactionHistoryBaseFee": {
-    "message": "Base fee (GWEI)"
+    "message": "Base Fee (GWEI)"
```
Comment (mcmire) [inline app/_locales/en/messages.json:2567]: Opted to titlecase all of the properties in the transaction breakdown to keep things consistent.

File: ui/components/ui/currency-display/currency-display.component.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ CurrencyDisplay.propTypes = {
   'className': PropTypes.string,
   'currency': PropTypes.string,
   'data-testid': PropTypes.string,
-  'denomination': PropTypes.oneOf([GWEI]),
+  'denomination': PropTypes.oneOf([GWEI, ETH]),
```
Comment (mcmire) [inline ui/components/ui/currency-display/currency-display.component.js:56]: For some reason I was finding after adding max fee per gas that I was getting an error about ETH not being a valid denomination. Adding ETH to the list fixed this issue, but I am curious whether there is a better way to solve this problem as I do not totally understand how this works.

**Review by brad-decker — DISMISSED** [review]
LGTM

File: ui/components/ui/currency-display/currency-display.component.js (hunk 2 above)
Comment (adonesky1) [reply ui/components/ui/currency-display/currency-display.component.js:56]: Yeah that's very odd given that there are other UserPreferencedCurrencyDisplay instances getting passed `ETH` as denomination in the same component...

Comment (adonesky1) [reply ui/components/ui/currency-display/currency-display.component.js:56]: That being said this should be included here. So I guess propTypes is as unreliable as we've always assumed

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Review by brad-decker — APPROVED** [review]

File: ui/components/ui/currency-display/currency-display.component.js (hunk 2 above)
Comment (mcmire) [reply ui/components/ui/currency-display/currency-display.component.js:56]: Indeed :)

**Pull request merged**